README.md

```
@@ -1,42 +1,295 @@
+
+| Adapter | Configuration |
+|---------|---------------|
+| Console (dry-run) | `trade.ConsoleAdapter()` |
+| Alpaca | `trade.Alpaca(new AlpacaOptions { UsePaper = true })` |
+
+For migration from v1.x, see [MIGRATION.md](MIGRATION.md).
+
+## Dev Console
+A developer console is available to run batch, streaming, and multi-series examples locally.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)